src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -88,7 +88,7 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
 
     const numSvc = cy.$(`node[nodeType = "${NodeType.SERVICE}"]`).size();
     const numWorkloads = cy.$(`node[nodeType = "${NodeType.WORKLOAD}"]`).size();
-    const numApps = cy.$(`node[nodeType = "${NodeType.APP}"][!isGroup]`).size();
+    const { numApps, numVersions } = this.countApps(cy);
```

Comment:
And suddenly I realize that javascript functions can return multiple args :)

Reply:
Technically, it can only return one arg -- in this case, an object. You "unwrap" that object with this syntax.

But, honestly, I also prefer to think that it's returning multiple args :)